client/src/helpers/index.js

```diff
@@ -238,3 +259,224 @@ export const getDayTimeNow = () => {
 
   return [dayNow, timeTime];
 };
+
+export const getDayOfWeekNum = (dayOfWeekString) => {
```

Comment:
Nit: reference the previous comment, we can simplify the function using an object to map the day of the week to number counterparts.